.github/workflows/pull_request.yaml

```diff
@@ -23,4 +23,22 @@ jobs:
            echo "Validating template: $template"
            sam validate --template-file "$template" --region us-east-2 --lint
           done        
-        shell: bash
\ No newline at end of file
+        shell: bash
+
+  security:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Run CFN Nag Security Checks
+        uses: stelligent/cfn_nag@master
+        with:
+          input_path: .
+          extra_args: -o sarif
+          output_path: cfn_nag.sarif
+      - name: Upload CFN Nag SARIF
+        uses: github/codeql-action/upload-sarif@v3
```

Comment:
we are using @v4 for checkout, but @v3 for this. It is intended or can be changed to have same version everywhere ?

Reply:
That depends on Individual version release. 
latest version in checkout is v4v where as in upload serif is v3